style.css

```diff
@@ -0,0 +1,322 @@
+:root {
```

Comment:
The changes mostly appear copy pasted from the tutorial.

Missing rules for responsiveness.